datascience/tables.py

```diff
@@ -926,15 +1020,26 @@ def hist(self, overlay=False, bins=None, counts=None, **vargs):
         if overlay:
             # vargs.setdefault('histtype', 'stepfilled')
             plt.figure(figsize=(6, 4))
-            plt.hist(columns.values(), color=colors, **vargs)
+            (vals,rbins,patches) = plt.hist(list(columns.values()), color=colors, **vargs)
```

Comment:
How about we have separate methods for binning to create a table and drawing a histogram, rather than gluing them together like matplotlib. I think you usually either want to see the histogram or compute with it, but not very often both.